Add `Old Map Fantasy World Cup` tournament article (#6057)
ppy/osu-wiki: wiki/Tournaments/OMF/WC/en.md

- NoMod
  - [IOSYS - Okuu's Nuclear Fusion Dojo (Mafiamaster) \[v2b's Insane\]](https://osu.ppy.sh/beatmapsets/8442#osu/37166)
  - [Various Artists - Muya's Ancient beatmap Package (Muya) \[Suzaku - CONTRACT 2012\]](https://osu.ppy.sh/beatmapsets/1485589#osu/3046323)
  - [Hatsune Miku - With a Dance Number (val0108) \[0108 style\]](https://osu.ppy.sh/beatmapsets/29691#osu/98415)
  - [Takeharu Ishimoto - Twister (Subarashiki Mix) (awp) \[Ultimate\]](https://osu.ppy.sh/beatmapsets/1830#osu/17459)
  - [Wada Kouji - Hirari (Ekaru) \[Insane\]](https://cdn.discordapp.com/attachments/414934337981644811/878710381734932550/Wada_Kouji_-_Hirari.osz)
- Hidden
  - [Hirano Aya - Super Driver (CDFA) \[Collab\]](https://osu.ppy.sh/beatmapsets/21499#osu/74652)
  - [fripSide - Assemble*LOVEsemble (ykcarrot) \[Ensemble\]](https://osu.ppy.sh/beatmapsets/10787#osu/42499)
  - [TERRA - We Are (James) \[Insane\]](https://osu.ppy.sh/beatmapsets/846#osu/5413)
- HardRock
  - [IOSYS - Chanteikku Sanyousei no Itazura Daisensou (Kurosanyan) \[Strongest Fairy\]](https://osu.ppy.sh/beatmapsets/29307#osu/100392)
  - [Cranky - Luvin' You (avira0922) \[MAX\]](https://osu.ppy.sh/beatmapsets/8923#osu/36134)

> - Doubletime HardRock

Reviewer: ```suggestion
- DoubleTime & HardRock
```

  - [Mame - Aka ni Somaru Dress (Mystica) \[Lolita\]](https://osu.ppy.sh/beatmapsets/33217#osu/108387)
- Flashlight
  - [BRIGHT - Ichinen Nikagetsu Hatsuka (Nymph) \[Hard\]](https://osu.ppy.sh/beatmapsets/50270#osu/154582)
  - [Santana feat. Rob Thomas - Smooth (lesjuh) \[Hard\]](https://osu.ppy.sh/beatmapsets/9781#osu/38695)
- Tiebreaker
  - [Ayane - Endless Tears... (lkp) \[Plus\]](https://osu.ppy.sh/beatmapsets/24960#osu/85355)

### Qualifiers

- NoMod
  - [Susumu Hirasawa - The Secret of The Flowers of Phenomenon (jesus1412) \[Insane\]](https://osu.ppy.sh/beatmapsets/70247#osu/201974)
  - [O-Life Japan - Yakujinsama no Couple Dance (AngelHoney) \[Lunatic\]](https://osu.ppy.sh/beatmapsets/16990#osu/95954)
  - [Chata - Remind (Patchouli) \[Overdrive!!\]](https://osu.ppy.sh/beatmapsets/37292#osu/120030)
- Hidden
  - [Horie Yui - Coloring (IchiGUN) \[Color of Heart\]](https://osu.ppy.sh/beatmapsets/44433#osu/166749)
  - [Ruzarin Kashiwagi - Theme of Love (Short Ver.) (Hitoshirenu Shourai) \[Insane\]](https://osu.ppy.sh/beatmapsets/3892#osu/22828)
- HardRock
  - [Kola Kid - can't hide your love (Kert) \[Can't\]](https://osu.ppy.sh/beatmapsets/39732#osu/126446)

> - Doubletime HardRock

Reviewer: ```suggestion
- DoubleTime & HardRock
```

  - [Angela Chang - Adventure of True Love (James) \[Hard\]](https://osu.ppy.sh/beatmapsets/3301#osu/21428)
- DoubleTime

>   - [Mina - Kiss Kiss (osu_CN_yaya) \[Another\]](https://osu.ppy.sh/beatmapsets/9658#osu/38359)

Reviewer: ```suggestion
  - [Mina - Kiss Kiss (osu\_CN\_yaya) \[Another\]](https://osu.ppy.sh/beatmapsets/9658#osu/38359)
```

  - [Midnight Project - Crystal Cinderella (C R E A M) \[Cookietic\]](https://osu.ppy.sh/beatmapsets/28017#osu/96708)
- Easy
  - [An - AxiVid (Nakagawa-Kanon) \[Another\]](https://osu.ppy.sh/beatmapsets/66000#osu/192802)
- Flashlight
  - [Pendulum - The Vulture (La Cataline) \[Insane\]](https://osu.ppy.sh/beatmapsets/24163#osu/82249)

## Match results

### Round of 32

> The stats of the Round of 32 can be found in [this spreadsheet](https://docs.google.com/spreadsheets/d/1dpX2983pjfa6FFzycWS7E_sEEd2rrH_ogVMxhJ1ibDI/edit?usp=sharing "Google spreadsheet").

Reviewer: ```suggestion
The stats of the Round of 32 can be found in [this spreadsheet](https://docs.google.com/spreadsheets/d/1dpX2983pjfa6FFzycWS7E_sEEd2rrH_ogVMxhJ1ibDI/edit?usp=sharing "Google Sheets").
```

| **Hungary** ![][flag_HU] | **-1** | 5 | ![][flag_ES] Spain |  |
| **Russian Federation B** ![][flag_RU] | **5** | 4 | ![][flag_AR] Argentina | [#1](https://osu.ppy.sh/community/matches/90073561) |
| **Austria B** ![][flag_AT] | **4** | 5 | ![][flag_PT] Portugal | [#1](https://osu.ppy.sh/community/matches/90076404) |
| **Countryless B** ![][flag___] | **-1** | 5 | ![][flag_SK] Slovakia | [#1](https://osu.ppy.sh/community/matches/90079797) |
| **United Kingdom B** ![][flag_GB] | **1** | 5 | ![][flag_PH] Philippines | [#1](https://osu.ppy.sh/community/matches/90065138) |
| **Colombia** ![][flag_CO] | **3** | 5 | ![][flag_DK] Denmark | [#1](https://osu.ppy.sh/community/matches/90080188) |

Monday, 30 August 2021:

| Team 1 |  |  | Team 2 | Match link |
| --: | :-: | :-: | :-- | :-- |
| **Finland B** ![][flag_FI] | **5** | 4 | ![][flag_EE] Estonia | [#1](https://osu.ppy.sh/community/matches/90128921) |
| **United Kingdom** ![][flag_GB] | **5** | 0 | ![][flag_IT] Italy | [#1](https://osu.ppy.sh/community/matches/90134682) |
| **Peru B** ![][flag_PE] | **0** | 5 | ![][flag_CZ] Czechia | [#1](https://osu.ppy.sh/community/matches/90139842) |

### Round of 64

> The stats of the Round of 64 can be found in [this spreadsheet](https://docs.google.com/spreadsheets/d/1qdjdu2Ppg_vZgl4DBBDaO2QA4mbsSLQiUefPr4j7jq8/edit#gid=1455419690 "Google spreadsheet").

Reviewer: ```suggestion
The stats of the Round of 64 can be found in [this spreadsheet](https://docs.google.com/spreadsheets/d/1qdjdu2Ppg_vZgl4DBBDaO2QA4mbsSLQiUefPr4j7jq8/edit#gid=1455419690 "Google Sheets").
```

| Statistician |  ![][flag_FR] [Kaeldori](https://osu.ppy.sh/users/962519) |
| Designer |  ![][flag_DE] [Suilerua](https://osu.ppy.sh/users/13137699) |

## Links

- [Discussion thread](https://osu.ppy.sh/community/forums/topics/1359817)
- [Livestream](https://www.twitch.tv/Kaeldori "Twitch")
- [Challonge bracket](https://challonge.com/OMFWC "Challonge")
- [Pick'ems page](https://pickem.hwc.hr/tournaments/72 "hwchr's pick'em") hosted by ![][flag_DE] [hallowatcher](https://osu.ppy.sh/users/1874761)
- **[Main sheet](https://docs.google.com/spreadsheets/d/1YwFNdAnKQL4jCqZQmxUGc-Q_5hHj1fdhOdbTcWKZhsw/edit#gid=0 "Google Sheets")**

## Participants

|  | Country | Members |
| :-: | :-: | :-- |

> | ![][flag_AR] | **Argentina** |  **[kaoshii](https://osu.ppy.sh/users/7807935)**, [Penguo](https://osu.ppy.sh/users/4389490), [Vexion](https://osu.ppy.sh/users/10361419), [Razekon](https://osu.ppy.sh/users/4700864), [Rimi](https://osu.ppy.sh/users/5194834), [Emiru Ikuno 2](https://osu.ppy.sh/users/9393446), [un perro](https://osu.ppy.sh/users/6573651), [Zaqlev](https://osu.ppy.sh/users/3188703) |

Reviewer 2: @TPGPL one space right? (whole table)

```suggestion
| ![][flag_AR] | **Argentina** | **[kaoshii](https://osu.ppy.sh/users/7807935)**, [Penguo](https://osu.ppy.sh/users/4389490), [Vexion](https://osu.ppy.sh/users/10361419), [Razekon](https://osu.ppy.sh/users/4700864), [Rimi](https://osu.ppy.sh/users/5194834), [Emiru Ikuno 2](https://osu.ppy.sh/users/9393446), [un perro](https://osu.ppy.sh/users/6573651), [Zaqlev](https://osu.ppy.sh/users/3188703) |
```

  - [Ruzarin Kashiwagi - Theme of Love (Short Ver.) (Hitoshirenu Shourai) \[Insane\]](https://osu.ppy.sh/beatmapsets/3892#osu/22828)
- HardRock
  - [Kola Kid - can't hide your love (Kert) \[Can't\]](https://osu.ppy.sh/beatmapsets/39732#osu/126446)
- Doubletime HardRock
  - [Angela Chang - Adventure of True Love (James) \[Hard\]](https://osu.ppy.sh/beatmapsets/3301#osu/21428)
- DoubleTime
  - [Mina - Kiss Kiss (osu_CN_yaya) \[Another\]](https://osu.ppy.sh/beatmapsets/9658#osu/38359)
  - [Midnight Project - Crystal Cinderella (C R E A M) \[Cookietic\]](https://osu.ppy.sh/beatmapsets/28017#osu/96708)
- Easy
  - [An - AxiVid (Nakagawa-Kanon) \[Another\]](https://osu.ppy.sh/beatmapsets/66000#osu/192802)
- Flashlight
  - [Pendulum - The Vulture (La Cataline) \[Insane\]](https://osu.ppy.sh/beatmapsets/24163#osu/82249)

## Match results

### Round of 32

The stats of the Round of 32 can be found in [this spreadsheet](https://docs.google.com/spreadsheets/d/1dpX2983pjfa6FFzycWS7E_sEEd2rrH_ogVMxhJ1ibDI/edit?usp=sharing "Google spreadsheet").

Saturday, 28 August 2021:
| Team 1 |  |  | Team 2 | Match link |
| --: | :-: | :-: | :-- | :-- |

> | **China** ![][flag_CN] | **5** | 4 | ![][flag_TH] Thailand | [#1](https://osu.ppy.sh/community/matches/90000968) |)
> | **Singapore B** ![][flag_SG] | **2** | 5 | ![][flag_HK] Hong Kong B | [#1](https://osu.ppy.sh/community/matches/90006529) |)
> | **Taiwan** ![][flag_TW] | **5** | 0 | ![][flag_FR] France B | [#1](https://osu.ppy.sh/community/matches/90006521) |)
> | **South Korea** ![][flag_KR] | **5** | 3 | ![][flag_BR] Brazil B | [#1](https://osu.ppy.sh/community/matches/90010297) |)
> | **Russian Federation** ![][flag_RU] | **5** | 0 | ![][flag_RO] Romania | [#1](https://osu.ppy.sh/community/matches/90010059) |)
> | **Hong Kong** ![][flag_HK] | **5** | 0 | ![][flag_AT] Austria | [#1](https://osu.ppy.sh/community/matches/90012374) |)
> | **Brazil** ![][flag_BR] | **5** | 1 | ![][flag_JP] Japan | [#1](https://osu.ppy.sh/community/matches/90012550) |)
> | **Poland** ![][flag_PL] | **5** | 4 | ![][flag_PL] Poland B | [#1](https://osu.ppy.sh/community/matches/90015402) |)
> | **United States** ![][flag_US] | **5** | 0 | ![][flag_SE] Sweden | [#1](https://osu.ppy.sh/community/matches/90022369) |)
> | **Canada** ![][flag_CA] | **5** | 2 | ![][flag_DE] Germany | [#1](https://osu.ppy.sh/community/matches/90025562) |)
> | **France** ![][flag_FR] | **5** | -1 | ![][flag_CL] Chile |  |)
> | **United States B** ![][flag_US] | **5** | 0 | ![][flag_CA] Canada B | [#1](https://osu.ppy.sh/community/matches/90034080) |)

Reviewer 2: same for all below tables

```suggestion
| **China** ![][flag_CN] | **5** | 4 | ![][flag_TH] Thailand | [#1](https://osu.ppy.sh/community/matches/90000968) |
| **Singapore B** ![][flag_SG] | **2** | 5 | ![][flag_HK] Hong Kong B | [#1](https://osu.ppy.sh/community/matches/90006529) |
| **Taiwan** ![][flag_TW] | **5** | 0 | ![][flag_FR] France B | [#1](https://osu.ppy.sh/community/matches/90006521) |
| **South Korea** ![][flag_KR] | **5** | 3 | ![][flag_BR] Brazil B | [#1](https://osu.ppy.sh/community/matches/90010297) |
| **Russian Federation** ![][flag_RU] | **5** | 0 | ![][flag_RO] Romania | [#1](https://osu.ppy.sh/community/matches/90010059) |
| **Hong Kong** ![][flag_HK] | **5** | 0 | ![][flag_AT] Austria | [#1](https://osu.ppy.sh/community/matches/90012374) |
| **Brazil** ![][flag_BR] | **5** | 1 | ![][flag_JP] Japan | [#1](https://osu.ppy.sh/community/matches/90012550) |
| **Poland** ![][flag_PL] | **5** | 4 | ![][flag_PL] Poland B | [#1](https://osu.ppy.sh/community/matches/90015402) |
| **United States** ![][flag_US] | **5** | 0 | ![][flag_SE] Sweden | [#1](https://osu.ppy.sh/community/matches/90022369) |
| **Canada** ![][flag_CA] | **5** | 2 | ![][flag_DE] Germany | [#1](https://osu.ppy.sh/community/matches/90025562) |
| **France** ![][flag_FR] | **5** | -1 | ![][flag_CL] Chile |  |
| **United States B** ![][flag_US] | **5** | 0 | ![][flag_CA] Canada B | [#1](https://osu.ppy.sh/community/matches/90034080) |
```

---
tags:
  - OMF
  - OMFWC
---

# Old Map Fantasy World Cup

![OMF World Cup Banner](img/banner.png)

> The **Old Map Fantasy World Cup** (***OMFWC***) is a country-based osu! tournament hosted by Kaeldori.

Reviewer 3: This should also say what iteration of OMF this is but I am not familiar enough with this tournament series to add it.

```suggestion
The **Old Map Fantasy World Cup** (***OMFWC***) is a country-based 4v4 double-elimination osu! tournament hosted by ![][flag_FR] [Kaeldori](https://osu.ppy.sh/users/962519).
```

Author: Well it's kinda a spin off in the series because normally it's a rank restricted tournament where anyone can signup

Reviewer 3: Hmm, I see. If that is the case this should be ```suggestion
The **Old Map Fantasy World Cup** (***OMFWC***) is a country-based 4v4 double-elimination osu! tournament hosted by ![][flag_FR] [Kaeldori](https://osu.ppy.sh/users/962519). It is the first iteration of the Old Map Fantasy World Cup.
```

> The **Old Map Fantasy World Cup** (***OMFWC***) is a country-based 4v4 double-elimination osu! tournament hosted by ![][flag_FR] [Kaeldori](https://osu.ppy.sh/users/962519).

Reviewer 3: ```suggestion
The **Old Map Fantasy World Cup** (***OMFWC***) is a country-based 4v4 double-elimination osu! tournament hosted by ![][flag_FR] [Kaeldori](https://osu.ppy.sh/users/962519). It is the first iteration of the Old Map Fantasy World Cup.
```

## Tournament schedule

| Event | Timestamp |
| --: | :-- |
| Registration phase | 2021-06-28/2021-08-08 |
| Qualifier stage | 2021-08-14/2021-08-16 |
| Round of 64 | 2021-08-21/2021-08-22 |
| Round of 32 | 2021-08-28/2021-08-29 |
| Round of 16 | 2021-09-04/2021-09-05 |
| Quarterfinals | 2021-09-11/2021-09-12 |
| Semifinals | 2021-09-18/2021-09-19 |
| Finals | 2021-09-25/2021-09-26 |
| Grand Finals | 2021-10-02/2021-10-03 |

## Prizes

| Placing | Prize(s) |
| :-: | :-- |
| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | Profile badge, profile banner |
| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | Profile banner |
| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | Profile banner |

## Organisation

> The osu! World Cup 2020 was run by various community members.

Reviewer 3: ```suggestion
The Old Map Fantasy World Cup was run by various community members.
```

| Commentator | ![][flag_FR] [Kaeldori](https://osu.ppy.sh/users/962519), ![][flag_AU] [xtremeities](https://osu.ppy.sh/users/10759664), ![][flag_US] [Daprin](https://osu.ppy.sh/users/10961983), ![][flag_KR] [isome](https://osu.ppy.sh/users/87546), ![][flag_RU] [Prade](https://osu.ppy.sh/users/9318565), ![][flag_GB] [Doomsday](https://osu.ppy.sh/users/18983), ![][flag_NL] [n0ah](https://osu.ppy.sh/users/3086393) |
| Referee | ![][flag_ES] [Deif](https://osu.ppy.sh/users/318565), ![][flag_SG] [uchuuj1n](https://osu.ppy.sh/users/9140302), ![][flag_US] [Rekunan](https://osu.ppy.sh/users/8203119), ![][flag_VN] [rock-on](https://osu.ppy.sh/users/9676089), ![][flag_PL] [P a t r i c k](https://osu.ppy.sh/users/6814521), ![][flag_DE] [GDLenny](https://osu.ppy.sh/users/8406711), ![][flag_SI] [Redavor](https://osu.ppy.sh/users/3328606), ![][flag_PL] [Kondi](https://osu.ppy.sh/users/7382321), ![][flag_US] [freddiiieeee](https://osu.ppy.sh/users/7112839), ![][flag_KZ] [brwn31](https://osu.ppy.sh/users/10379418) |
| Statistician |  ![][flag_FR] [Kaeldori](https://osu.ppy.sh/users/962519) |

> | GFX |  ![][flag_DE] [Suilerua](https://osu.ppy.sh/users/13137699) |

Reviewer 3: ```suggestion
| Designer |  ![][flag_DE] [Suilerua](https://osu.ppy.sh/users/13137699) |
```

## Links

- [Discussion thread](https://osu.ppy.sh/community/forums/topics/1359817)
- [Livestream](https://www.twitch.tv/Kaeldori "Twitch")
- [Challonge bracket](https://challonge.com/OMFWC "Challonge")
- [Pick'ems page](https://pickem.hwc.hr/tournaments/72 "hwchr's pick'em") hosted by ![][flag_DE] [hallowatcher](https://osu.ppy.sh/users/1874761)
- **[Main sheet](https://docs.google.com/spreadsheets/d/1YwFNdAnKQL4jCqZQmxUGc-Q_5hHj1fdhOdbTcWKZhsw/edit#gid=0 "Google Sheets")**

> ## Participants

Reviewer 3: My assumption here is that the participants list is sorted by seed rather than by alphabetical order. My preference would be to change the table to be in alphabetical order to match with other would cup style tournaments.

Author: Fixed

- HardRock
  - [Rohi - Kodoku Egoism (NatsumeRin) \[Nold\]](https://osu.ppy.sh/beatmapsets/58737#osu/185667)
  - [DM Ashura - Chaotic WHITE (AngelHoney) \[Chaos\]](https://osu.ppy.sh/beatmapsets/22874#osu/81304)
- Doubletime HardRock
  - [Matchbox Twenty - Push (mrtn) \[Hard\]](https://osu.ppy.sh/beatmapsets/4047#osu/23207)
- DoubleTime
  - [SuganoMusic - Imademo... (S i R i R u) \[Lunatic\]](https://osu.ppy.sh/beatmapsets/21960#osu/75930)
  - [DJ YOSHITAKA - ALBIDA (dksslqj) \[gow's Advanced\]](https://osu.ppy.sh/beatmapsets/19561#osu/75266)
  - [Utada Hikaru - Prisoner Of Love (DawnII) \[Insane\]](https://osu.ppy.sh/beatmapsets/1781#osu/21407)
- Easy
  - [IOSYS - Fragrance of Oriental Sunflower (Regou) \[Lunatic\]](https://osu.ppy.sh/beatmapsets/49975#osu/153835)
  - [Nekomata Master - Byakuya Gentou (ignorethis) \[EX\]](https://osu.ppy.sh/beatmapsets/36849#osu/119375)
- Flashlight
  - [Getter Jaani - Rockefeller Street (Nightcore Mix) (osuplayer111) \[Andrea\]](https://osu.ppy.sh/beatmapsets/33842#osu/110175)
  - [TeddyLoid - Theme for Scanty & Knee Socks (ztrot) \[Heavy\]](https://osu.ppy.sh/beatmapsets/24886#osu/114332)
- Tiebreaker

>   - [Susumu Hirasawa - Night Walking Wearing the Human Body (jesus1412) \[Insane\]](https://osu.ppy.sh/beatmapsets/60938#osu/181975)

Reviewer 3: ```suggestion
  - **[Susumu Hirasawa - Night Walking Wearing the Human Body (jesus1412) \[Insane\]](https://osu.ppy.sh/beatmapsets/60938#osu/181975)**
```

- HardRock
  - [IOSYS - Chanteikku Sanyousei no Itazura Daisensou (Kurosanyan) \[Strongest Fairy\]](https://osu.ppy.sh/beatmapsets/29307#osu/100392)
  - [Cranky - Luvin' You (avira0922) \[MAX\]](https://osu.ppy.sh/beatmapsets/8923#osu/36134)
- Doubletime HardRock
  - [BeForU - Moonbike (James) \[Hard\]](https://osu.ppy.sh/beatmapsets/1505#osu/13250)
- DoubleTime
  - [Starving Trancer feat. MAKI - only my railgun (Leorda) \[Insane\]](https://osu.ppy.sh/beatmapsets/43513#osu/137575)
  - [Suzaku - Anisakis -somatic mutation type''Forza''- (ouranhshc) \[RLC's Another\]](https://osu.ppy.sh/beatmapsets/60521#osu/882052)
  - [Lou Bega - Mambo No.5 (Lesjuh) \[Insane\]](https://osu.ppy.sh/beatmapsets/33393#osu/108936)
- Easy
  - [Lily-an - Koisuru Recipe (Holoaz) \[Lunatic\]](https://osu.ppy.sh/beatmapsets/69173#osu/199686)
  - [Rabpit - Sanctity (tsuka) \[Expert\]](https://osu.ppy.sh/beatmapsets/50017#osu/153926)
- Flashlight
  - [Brandy - Visual Dream !! (yaya) \[DaRRi MIx\]](https://osu.ppy.sh/beatmapsets/13956#osu/51300)
  - [Far East Movement ft. The Cataracs - Like a G6 (poisonchan) \[Like a Katsu\]](https://osu.ppy.sh/beatmapsets/21922#osu/92835)
- Tiebreaker

>   - [NicoNicoDouga - Nanairo no Nico Nico Douga (Doomsday) \[2009 Marathon\]](https://osu.ppy.sh/beatmapsets/16831#osu/1551754)

Reviewer 3: ```suggestion
  - **[NicoNicoDouga - Nanairo no Nico Nico Douga (Doomsday) \[2009 Marathon\]](https://osu.ppy.sh/beatmapsets/16831#osu/1551754)**
```

- HardRock
  - [Kagamine Rin - Tokyo Teddy Bear (NatsumeRin) \[Rin\]](https://osu.ppy.sh/beatmapsets/39838#osu/127138)
  - [Miki Sayaka vs. Miki Sayaka (fw. Miki Sayaka) - squartatrice (Muya) \[Another\]](https://osu.ppy.sh/beatmapsets/40368#osu/131009)
- Doubletime HardRock
  - [Wizet. Nexon - Coke Town (Metroid) \[Hard\]](https://osu.ppy.sh/beatmapsets/5424#osu/26660)
- DoubleTime
  - [Megpoid GUMI - Mozaik Role (NatsumeRin) \[PoLKs\]](https://osu.ppy.sh/beatmapsets/20817#osu/74779)
  - [iconoclasm - perditus paradisus (Sharp) \[Hyper\]](https://osu.ppy.sh/beatmapsets/22212#osu/83343)
  - [Comp - Gensou no Satellite (Shinxyn) \[Insane (OMF Edit)\]](https://osu.ppy.sh/beatmapsets/4404#osu/63604)
- Easy
  - [Hatsune Miku - Toosenbo (NatsumeRin) \[Rin\]](https://osu.ppy.sh/beatmapsets/18266#osu/65429)
  - [Mame - Aka ni Somaru Dress (Mystica) \[Lolita\]](https://osu.ppy.sh/beatmapsets/33217#osu/108387)
- Flashlight
  - [BRIGHT - Ichinen Nikagetsu Hatsuka (Nymph) \[Hard\]](https://osu.ppy.sh/beatmapsets/50270#osu/154582)
  - [Santana feat. Rob Thomas - Smooth (lesjuh) \[Hard\]](https://osu.ppy.sh/beatmapsets/9781#osu/38695)
- Tiebreaker

>   - [Ayane - Endless Tears... (lkp) \[Plus\]](https://osu.ppy.sh/beatmapsets/24960#osu/85355)

Reviewer 3: ```suggestion
  - **[Ayane - Endless Tears... (lkp) \[Plus\]](https://osu.ppy.sh/beatmapsets/24960#osu/85355)**
```

  - [O-Life Japan - Yakujinsama no Couple Dance (AngelHoney) \[Lunatic\]](https://osu.ppy.sh/beatmapsets/16990#osu/95954)
  - [Chata - Remind (Patchouli) \[Overdrive!!\]](https://osu.ppy.sh/beatmapsets/37292#osu/120030)
- Hidden
  - [Horie Yui - Coloring (IchiGUN) \[Color of Heart\]](https://osu.ppy.sh/beatmapsets/44433#osu/166749)
  - [Ruzarin Kashiwagi - Theme of Love (Short Ver.) (Hitoshirenu Shourai) \[Insane\]](https://osu.ppy.sh/beatmapsets/3892#osu/22828)
- HardRock
  - [Kola Kid - can't hide your love (Kert) \[Can't\]](https://osu.ppy.sh/beatmapsets/39732#osu/126446)
- Doubletime HardRock
  - [Angela Chang - Adventure of True Love (James) \[Hard\]](https://osu.ppy.sh/beatmapsets/3301#osu/21428)
- DoubleTime
  - [Mina - Kiss Kiss (osu_CN_yaya) \[Another\]](https://osu.ppy.sh/beatmapsets/9658#osu/38359)
  - [Midnight Project - Crystal Cinderella (C R E A M) \[Cookietic\]](https://osu.ppy.sh/beatmapsets/28017#osu/96708)
- Easy
  - [An - AxiVid (Nakagawa-Kanon) \[Another\]](https://osu.ppy.sh/beatmapsets/66000#osu/192802)
- Flashlight
  - [Pendulum - The Vulture (La Cataline) \[Insane\]](https://osu.ppy.sh/beatmapsets/24163#osu/82249)

## Match results

> 

Reviewer 3: ```suggestion
```

### Round of 32

> The stats of the Round of 32 can be found in [this spreadsheet](https://docs.google.com/spreadsheets/d/1Kj0Y1ycdQjPPGIBNUvJqnEi4U5UlBvJPhkXQbsmTYLg/edit?rm=minimal "Google spreadsheet").

Reviewer 3: Is this intended? Currently links to a OWC2020 spreadsheet?

Author: I failed my copypaste it seems, thanks you for noticing

| **Hungary** ![][flag_HU] | **-1** | 5 | ![][flag_ES] Spain | |)
| **Russian Federation B** ![][flag_RU] | **5** | 4 | ![][flag_AR] Argentina | [#1](https://osu.ppy.sh/community/matches/90073561) |)
| **Austria B** ![][flag_AT] | **4** | 5 | ![][flag_PT] Portugal | [#1](https://osu.ppy.sh/community/matches/90076404) |)
| **Countryless B** ![][flag_XX] | **-1** | 5 | ![][flag_SK] Slovakia | [#1](https://osu.ppy.sh/community/matches/90079797) |)
| **United Kingdom B** ![][flag_GB] | **1** | 5 | ![][flag_PH] Philippines | [#1](https://osu.ppy.sh/community/matches/90065138) |)
| **Colombia** ![][flag_CO] | **3** | 5 | ![][flag_DK] Denmark | [#1](https://osu.ppy.sh/community/matches/90080188) |)

Monday, 30 August 2021:
| Team 1 |  |  | Team 2 | Match link |
| --: | :-: | :-: | :-- | :-- |
| **Finland B** ![][flag_FI] | **5** | 4 | ![][flag_EE] Estonia | [#1](https://osu.ppy.sh/community/matches/90128921) |)
| **United Kingdom** ![][flag_GB] | **5** | 0 | ![][flag_IT] Italy | [#1](https://osu.ppy.sh/community/matches/90134682) |)
| **Peru B** ![][flag_PE] | **0** | 5 | ![][flag_CZ] Czechia | [#1](https://osu.ppy.sh/community/matches/90139842) |)

### Round of 64

> 

Reviewer 3: ```suggestion
```

> Saturday, 28 August 2021:

Reviewer 3: ```suggestion
Saturday, 28 August 2021:

```

| **Taiwan** ![][flag_TW] | **5** | 0 | ![][flag_IL] Israel | [#1](https://osu.ppy.sh/community/matches/89669002) |)
| **Netherlands** ![][flag_NL] | **5** | 0 | ![][flag_EE] Estonia | [#1](https://osu.ppy.sh/community/matches/89671085) |)
| **Norway** ![][flag_NO] | **2** | 5 | ![][flag_FR] France B | [#1](https://osu.ppy.sh/mp/89671687) |)
| **Brazil** ![][flag_BR] | **5** | 0 | ![][flag_SG] Singapore B | [#1](https://osu.ppy.sh/community/matches/89673897) |)
| **Germany** ![][flag_DE] | **5** | 3 | ![][flag_SK] Slovakia | [#1](https://osu.ppy.sh/community/matches/89676440) |)
| **Poland B** ![][flag_PL] | **5** | 0 | ![][flag_PT] Portugal | [#1](https://osu.ppy.sh/community/matches/89677184) |)
| **Italy** ![][flag_IT] | **5** | 1 | ![][flag_UY] Uruguay | [#1](https://osu.ppy.sh/community/matches/89679656) |)
| **Sweden** ![][flag_SE] | **5** | 1 | ![][flag_MX] Mexico | [#1](https://osu.ppy.sh/community/matches/89682748) |)
| **Romania** ![][flag_RO] | **5** | 2 | ![][flag_ES] Spain | [#1](https://osu.ppy.sh/community/matches/89683718) |)
| **Countryless** ![][flag_XX] | **1** | 5 | ![][flag_BR] Brazil B | [#1](https://osu.ppy.sh/community/matches/89685117) |)

> Monday, 30 August 2021:

Reviewer 3: ```suggestion
Monday, 30 August 2021:

```

| Team 1 |  |  | Team 2 | Match link |
| --: | :-: | :-: | :-- | :-- |
| **Russian Federation B** ![][flag_RU] | **5** | 0 | ![][flag_GB] United Kingdom B | [#1](https://osu.ppy.sh/community/matches/89731229) |)
| **Poland** ![][flag_PL] | **5** | 1 | ![][flag_AT] Austria B | [#1](https://osu.ppy.sh/community/matches/89733422) |)
| **Finland** ![][flag_FI] | **5** | 0 | ![][flag_FI] Finland B | [#1](https://osu.ppy.sh/community/matches/89733807) |)
| **Peru** ![][flag_PE] | **5** | 0 | ![][flag_NL] Netherlands B | [#1](https://osu.ppy.sh/community/matches/89750968) |)

### Qualifiers

> The stats of the Qualifiers can be found in [this spreadsheet](https://docs.google.com/spreadsheets/d/17yFrT7yEf5x01Xx3v4yxHNCEdOl630svCdFs2WIpRYk/edit#gid=809028813 "Google spreadsheet").

Reviewer 3: ```suggestion
See the [Qualifiers results spreadsheet](https://docs.google.com/spreadsheets/d/17yFrT7yEf5x01Xx3v4yxHNCEdOl630svCdFs2WIpRYk/edit#gid=809028813 "Google Sheets") for lists of all qualification results.
```
